circle.yml

```diff
@@ -7,8 +7,9 @@ machine:
 dependencies:
     override:
         - sudo pip install -U docker-compose
+        - sudo pip install -r tests/requirements.txt
 
 test:
     override:
         - docker-compose -f docker-compose-deploy.yml up -d
-        - curl -s -o /dev/null -w "%{http_code}" http://pay.dev:8000/api/
+        - python tests/status.py
```

Comment:
if you made this `nosetests` then it would be easier to add additional files later on. It will find the `tests` dir automatically.